src/Control/Distributed/Process/ManagedProcess/Internal/Types.hs

```diff
@@ -80,10 +81,11 @@ type CallId = MonitorRef
 
 newtype CallRef a = CallRef { unCaller :: (Recipient, CallId) }
   deriving (Eq, Show, Typeable, Generic)
-instance Serializable a => Binary (CallRef a) where
-instance NFData a => NFData (CallRef a) where rnf (CallRef x) = rnf x `seq` ()
+--instance Serializable a => Binary (CallRef a) where
```

Comment:
GHC 8 introduces warnings for unnecessary constraints. I should have left this chunk out of the pull request since it is non-essential, but please be aware that the GHC 8 generates a lot of new warnings for this project.